scripts/meetingbaas_api.py

```
@@ -0,0 +1,150 @@
+import logging
+from enum import Enum
+from typing import Any, Dict, List, Optional, Union
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import

(F401)